### ISSUE TITLE
Annotation marker hides annotations when time clashes

### ISSUE THREAD

**Issue by mrajah-twttr** [openzipkin/zipkin#3117]
## Describe the Bug
When a span has multiple annotations with the exact same timestamp the bubble circles that show them on a linear scale seem to overlap on top of each other. When you click on the bubble only the last annotation is shown masking the existing of other annotations at the same point.

## Steps to Reproduce
Create a trace with a span with multiple annotations for exact same timestamp

## Expected Behaviour
click on the the annotation bubble should show all annotations matching the timestamp and not just the element on the top.

**Comment by jorgheymans** [comment]
Thanks for reporting, any chance you can attach such trace and maybe a screenshot / animated gif showing the issue ? You can download the trace through the UI. Make sure to remove any sensitive data (easily done using search/replace from a text editor, as values are often repeated).

**Comment by mrajah-twttr** [comment]
thanks for your quick response. attached are screenshots that show missing bubbles (there are 3 circles visible but you can see atleast 4 listed in the show all annos view). i am not entirely sure what aspects of the data i can share yet, ideally this should be easy repro? i could see the DOM elements for each circle listing all the annos as well.[](url)
![image](https://user-images.githubusercontent.com/66497287/84825377-34af7e80-afd6-11ea-984f-97d5e7000d4f.png)
![image](https://user-images.githubusercontent.com/66497287/84825436-455ff480-afd6-11ea-995a-0d5c9bb7a9d1.png)

**Comment by jorgheymans** [comment]
Thanks for the screenshot. For sure it's easy repro but someone willing to
fix this bug would be happy to have at least one test case readily
available to work with. It just reduces the needed energy and threshold
when wanting to spend time on fixing this (hence increasing the odds of it
getting looked at).

**Comment by mrajah-twttr** [comment]
Makes sense.  ls there a standard test trace json i can use to create a repro ?

**Comment by jorgheymans** [comment]
https://github.com/openzipkin/zipkin/tree/master/zipkin-lens/testdata but i think easier would be to just anonymize an existing trace, whatever works for you though.

**Comment by mrajah-twttr** [comment]
Actually https://github.com/openzipkin/zipkin/blob/master/zipkin-lens/testdata/netflix.json shows this error. you can look at the 2nd span for service EVCACHE_G it has 3 annos but only 1 circle is shown since they all share same timestamp.
